apps/registry/codemods/eslint/biome/migrate-rules/src/index.ts

```diff
@@ -0,0 +1,215 @@
+import type { Filemod } from '@codemod-com/filemod';
+import { isNeitherNullNorUndefined } from '@codemod-com/utilities/functions/validationMethods';
```

Comment:
it sucks, but this was the only solution that worked for me in order to be able to tree-shake this file. it's supposed to be used for CJS modules, however we are bundling ESM, which should just work out of the box. tree-shaking for some reason is broken within the build system, i have tried various hacks and potential solutions, but so far this was the only one that worked.

Reply:
You could even copy the function, that's not a problem :)